### PR TITLE
Pagenation design

### PR DESCRIPTION
**Pull request by m4ximuel**
# Pagenation design [feat]
Applying a new design to pagination

# PR Checklist
- [x] I've started from master
- [x] I've only committed changes related to this PR
- [x] All Unit tests pass
- [x] I've removed all commented code
- [x] I've removed all unneeded console.log statements

### CODE REVIEW

**Comment by m4ximuel** [comment]
pagenation

File: src/styles/styles.scss [hunk 1]
```diff
@@ -3,6 +3,46 @@
 
 $sideborder-margin: 6px;
 
+.selMode.done {
```
Comment (WDaan) [inline src/styles/styles.scss:6]: I don't think these are necessary :p

Comment (m4ximuel) [reply src/styles/styles.scss:6]: I start from the master, but I don't know why it keeps adding this. X(

**Comment by m4ximuel** [comment]
@WDaan Um...I deleted it manually once, is it okay with this method?

**Comment by m4ximuel** [comment]
#249

**Pull request closed without merging**